cms/templates/js/course-outline.underscore

```diff
@@ -200,6 +189,14 @@ if (is_proctored_exam) {
                     </p>
                 </div>
             <% } %>
+            <div class="status-hide-after-due">
+                <p>
+                    <% if (xblockInfo.get('hide_after_due')) { %>
+                        <span class="icon fa fa-eye-slash" aria-hidden="true"></span>
+                        <span class="status-hide-after-due-value"> <%- gettext("Subsection will remain hidden after due date") %> </span>
```

Comment:
I'm not sure. @catong What do you think? The assignment type is more specific, but in my mind the user is already thinking in a section-subsection-unit level when this string is viewed.

![screen shot 2016-07-19 at 10 58 56 am](https://cloud.githubusercontent.com/assets/7373924/16954731/f3497604-4d9f-11e6-86d3-d6f2927db964.png)